lighthouse-core/formatters/partials/table.css

```diff
@@ -33,6 +33,7 @@
   font-family: monospace;
   display: block;
   margin: 0;
+  overflow-y: hidden;
```

Comment:
`overflow-x: auto`. Tells the `pre` what to do (instead of telling it what not to do).
Would make me less nervous with tables, pre wrapping, and different browser compat in the mix. Clearly the latter is already an issue for FF.